.github/CODEOWNERS

```diff
@@ -64,7 +65,7 @@ launch/tier4_autoware_api_launch/* isamu.takagi@tier4.jp ryohsuke.mitsudome@tier
 launch/tier4_perception_launch/* yukihiro.saito@tier4.jp
 launch/tier4_planning_launch/* takamasa.horibe@tier4.jp
 launch/tier4_localization_launch/* yamato.ando@tier4.jp
-launch/tier4_simulator_launch/* keisuke.shima@tier4.jp
+launch/tier4_simulator_launch/** keisuke.shima@tier4.jp
```

Comment:
Test for https://github.com/autowarefoundation/autoware.universe/pull/1788.

No owner assigned.
![image](https://user-images.githubusercontent.com/31987104/190040686-763dca72-b83f-476e-b8bf-84fcbf199c08.png)

Reply:
Worked.
![image](https://user-images.githubusercontent.com/31987104/190041807-93242574-eda7-42bf-b327-498753f4b6db.png)